tests/test_wave_03.py

```diff
@@ -128,13 +128,13 @@ def test_mark_complete_missing_task(client):
     # Assert
     assert response.status_code == 404
 
-    raise Exception("Complete test with assertion about response body")
+    # raise Exception("Complete test with assertion about response body")
     # *****************************************************************
     # **Complete test with assertion about response body***************
     # *****************************************************************
```

Comment:
It looks like you missed actually updating this `test_mark_complete_missing_task` test as well as the `test_mark_incomplete_missing_task` test to have assertions checking the response bodies. If you hadn't shown other places in the tests where you _do_ write these types of assertions, I may have marked this submission Yellow.